build.gradle.kts

```diff
@@ -88,16 +99,20 @@ subprojects {
         // JxBrowser for Swing dependency.
         implementation(jxbrowser.swing)
 
+        // JxBrowser for Compose dependency.
+        implementation(jxbrowser.compose)
+
+        // Dependency on Compose for the current platform.
+        implementation(compose.desktop.currentOs)
+
         // JxBrowser for SWT dependency.
         implementation(jxbrowser.swt)
 
-        // Dependency on a SWT for the current platform.
+        // Dependency on an SWT for the current platform.
         implementation(Swt.toolkitDependency)
 
         // Depend on Guava for the Resources utility class used for loading resource files into strings.
         implementation("com.google.guava:guava:$guavaVersion")
-
-        implementation(files("$rootDir/examples/src/main/resources/resource.jar"))
```

Comment:
What was this?

Reply:
The file is used by one of the snippets, but it is not present anymore. Gradle fails such a build now.

I've created the issue: https://github.com/TeamDev-IP/JxBrowser-Docs/issues/921